src/apps/eth/eth.c

```diff
@@ -60,15 +69,54 @@ bool app_eth_address(
     }
```

Comment:
`return true` one line up looks wrong. possibly address verification does not work anymore because of the early abort here.